### PR TITLE
fix: Minor Android styling adjustments

### PR DESCRIPTION
**Pull request by jimhunty**
These include:
* removing capitalisation from the button
* Increasing the button font size
* Label to Headline bottom margin removed
* Button border radius reduced to 2

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/b5b3713e5a39b4a1ad491a622898109aac7e772d/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/b5b3713e5a39b4a1ad491a622898109aac7e772d/index.html

File: packages/button/__tests__/android/__snapshots__/button.android.test.js.snap [hunk 1]
```diff
@@ -23,14 +23,14 @@ exports[`1. button 1`] = `
...
         "color": "#FFFFFF",
         "fontFamily": "GillSansMTStd-Medium",
-        "fontSize": 14,
+        "fontSize": 16,
         "lineHeight": 14,
         "paddingTop": 0,
       }
     }
-    title="TEST BUTTON"
+    title="Test button"
```
Comment (craigbilner) [inline packages/button/__tests__/android/__snapshots__/button.android.test.js.snap]: this doesn't test `capitalise` behaviour?

Comment (colincclark) [reply packages/button/__tests__/android/__snapshots__/button.android.test.js.snap]: description says he removed capitalisation

File: packages/button/src/button.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
 import React from "react";
 import { Text, TouchableHighlight } from "react-native";
 import { colours } from "@times-components/styleguide";
-import transformTitle from "./utils";
```
Comment (colincclark) [inline packages/button/src/button.js]: can this file be deleted now?

Comment (jimhunty) [reply packages/button/src/button.js]: Thought I had done this. Now done.

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/a6ba57ca94d5ff381a481edc2e3620e56745d07d/index.html

File: packages/button/__tests__/android/__snapshots__/button.android.test.js.snap (hunk 1 above)
Comment (craigbilner) [reply packages/button/__tests__/android/__snapshots__/button.android.test.js.snap]: :speaking_head: meant removed `uppercase`, `capitalise` behaviour still needs testing

**Pull request merged**